edge_computing/image-base-install/ibi-preparing-for-image-based-install.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-preparing-for-image-based-install_{context}"]
+= Preparing for an image-based installation for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-understanding-image-based-install
+
+toc::[]
+To prepare for an image-based installation for {sno} clusters, you must complete the following tasks:
+
+* Install the {lcao} on the hub cluster.
```

Comment:
This should be removed, isn't a requirement

Reply:
Done

Reply:
@eranco74 - don't you need the lifecycle agent to create a seed image? And the seed image is a prerequisite for IBI?

Reply:
Updated the following line: Generate a seed image with LCA. Otherwise, the subsequent content talks about Installing the LCA and it's not clear why you would need to install it.